lib/storage/filesystem.js

```diff
@@ -49,22 +49,43 @@ function save(file, db, cb) {
 	if (writeQueue[file]) {
 		writeQueue[file].db = db;
 		writeQueue[file].callbacks.push(cb);
+        writeQueue[file].version += 1;
```

Comment:
can we use `writeQueue[file].version++`?

Reply:
+1